Doc/c-api/long.rst

```diff
@@ -571,6 +571,39 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
    .. versionadded:: 3.14
 
 
+.. c:function:: int PyLong_IsPositive(PyObject *obj)
+
+   Check if the integer object *obj* is positive.
```

Comment:
If you dislike strictly, I suggest:

```suggestion
   Check if the integer object *obj* is positive: greater than zero.
```

Reply:
I don't feel this addition is needed.

But at least this form is less confusing than "strictly positive".  Maybe: "Check if the integer object *obj* is positive, i.e. greater than zero."